### PR TITLE
Limit values of min max to <=100

### PR DESCRIPTION
**Pull request by doneill**
Fix to contrain percentage clip values.

### CODE REVIEW

**Comment by doneill** [comment]
@XuemingRocks Please review update to constrain percent clip parameters.

File: stretch-renderer/src/main/java/com/esri/arcgisruntime/sample/stretchrenderer/ParametersDialogFragment.java [hunk 1]
```diff
@@ -183,15 +182,20 @@ public void onStartTrackingTouch(SeekBar seekBar) { }
...
         updateSeekBar(mPercentClipMinSeekBar, mPercentClipMin, mCurrPercentClipMinTextView);
-        // move max to march min if max goes below min
-        if (mPercentClipMax < mPercentClipMin) {
-          mPercentClipMax = mPercentClipMin;
-          updateSeekBar(mPercentClipMaxSeekBar, mPercentClipMax, mCurrPercentClipMaxTextView);
+        if (mPercentClipMin + mPercentClipMax > 100) {
+            // constrain min + max <= 100
+            mPercentClipMax = 100 - mPercentClipMin;
+            updateSeekBar(mPercentClipMaxSeekBar, mPercentClipMax, mCurrPercentClipMaxTextView);
+        } else if (mPercentClipMax < mPercentClipMin){
```
Comment (XuemingRocks) [inline stretch-renderer/src/main/java/com/esri/arcgisruntime/sample/stretchrenderer/ParametersDialogFragment.java]: min and max define the percentile cutoff below and above respectively. Max is not necessarily > min. we should not put max > min as a constrain.

File: stretch-renderer/src/main/java/com/esri/arcgisruntime/sample/stretchrenderer/ParametersDialogFragment.java [hunk 2]
```diff
@@ -201,15 +205,20 @@ public void onStartTrackingTouch(SeekBar seekBar) { }
...
         updateSeekBar(mPercentClipMaxSeekBar, mPercentClipMax, mCurrPercentClipMaxTextView);
-        // move min to match max if min goes above max
-        if (mPercentClipMin > mPercentClipMax) {
-          mPercentClipMin = mPercentClipMax;
-          updateSeekBar(mPercentClipMinSeekBar, mPercentClipMin, mCurrPercentClipMinTextView);
+        if (mPercentClipMin + mPercentClipMax > 100) {
+            // constrain min + max <= 100
+            mPercentClipMin = 100 - mPercentClipMax;
+            updateSeekBar(mPercentClipMinSeekBar, mPercentClipMin, mCurrPercentClipMinTextView);
+        } else if (mPercentClipMin > mPercentClipMax){
```
Comment (XuemingRocks) [inline stretch-renderer/src/main/java/com/esri/arcgisruntime/sample/stretchrenderer/ParametersDialogFragment.java]: min and max define the percentile cutoff below and above respectively. Max is not necessarily > min. we should not put max > min as a constrain.

File: stretch-renderer/src/main/java/com/esri/arcgisruntime/sample/stretchrenderer/ParametersDialogFragment.java (hunk 1 above)
Comment (doneill) [inline stretch-renderer/src/main/java/com/esri/arcgisruntime/sample/stretchrenderer/ParametersDialogFragment.java]: @XuemingRocks Good point, I will remove the constraint.

**Comment by XuemingRocks** [comment]
@doneill looks good.

**Pull request merged**